src/Sylius/Bundle/AttributeBundle/Resources/config/doctrine/model/AttributeValue.orm.xml

```diff
@@ -21,7 +21,11 @@
             <generator strategy="AUTO" />
         </id>
 
-        <field name="value" column="value" type="text" nullable="true" />
+        <field name="text" column="text" type="text" nullable="true" />
+        <field name="boolean" column="boolean" type="boolean" nullable="true" />
+        <field name="integer" column="integer_value" type="integer" nullable="true" />
```

Comment:
There's a list of all restricted SQL keywords: https://www.drupal.org/node/141051

Can we stick to field name + `_value` convention also in text, boolean and datetime?